task/fossa-scanner/0.1/fossa-scanner.yaml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: fossa-scanner
```

Comment:
is there a way that this task can be tested?

Reply:
added test